pathod/pathod.py

```diff
@@ -284,10 +285,10 @@ def addlog(self, log):
         # moment because JSON encoding can't handle binary data, and I don't
         # want to base64 everything.
         if self.server.logreq:
-            encoded_bytes = self.rfile.get_log().encode("string_escape")
+            encoded_bytes = strutils.bytes_to_escaped_str(self.rfile.get_log())
```

Comment:
I'm not sure about these, since `bytes_to_escaped_str` returns a unicode string, and I think we need `bytes` here.